sbin/shipper_utils/update_and_install_workers.py

```diff
@@ -54,17 +55,23 @@ def update_docker_images(user, host, worker, autograding_workers, autograding_co
         for image in images_to_update:
             print(f"locally pulling the image '{image}'")
             try:
-              repo, tag = image.split(':')
-              client.images.pull(repository=repo, tag=tag)
+                repo, tag = image.split(':')
+                client.images.pull(repository=repo, tag=tag)
             except Exception as e:
-              print(f"ERROR: Could not pull {image}")
-              traceback.print_exc()
-              success = False
+                print(f"ERROR: Could not pull {image}")
+                traceback.print_exc()
+                success = False
+        docker_info = client.info()
```

Comment:
`client.info()` can throw a `docker.errors.APIError` exception, this is rare but could happen if the docker service isn't running correctly and should probably be handled